cmd/sebak/cmd/genesis.go

```diff
@@ -153,3 +178,63 @@ func makeGenesisBlock(genesisAddress, commonAddress, networkID, balanceStr, stor
 
 	return "", nil
 }
+
+func checkExistingAccounts(st *storage.LevelDBBackend, networkID, genesisAddress, commonAddress string, balance common.Amount) (created bool, err error) {
+	// check network id
+	var bt block.BlockTransaction
+	if bt, err = runner.GetGenesisTransaction(st); err != nil {
+		if err == errors.StorageRecordDoesNotExist {
+			created = false
+			err = nil
+			return
```

Comment:
I think `return false, nil` is better.

Reply:
@Charleslee522 I am still in love with named return :)

Reply:
This `created = false` code is redundant because created is already false.
Named return is okay :)